pkg/util/log/flags.go

```diff
@@ -65,6 +77,11 @@ func SetupRedactionAndStderrRedirects() (cleanup func(), err error) {
 	// won't be positioned outside of log redaction markers and
 	// mistakenly considered as "safe for reporting".
 
+	// Sanity check.
+	if IsActive() {
+		panic(errors.New("cannot set up logging after the first logging event"))
```

Comment:
Is this too specific to logging events? We could just leave it at it being "already active".